release-process.md

```diff
@@ -35,6 +35,34 @@ If you are a new Release Manager, you can read up on the process from the follow
 - gpg for signing https://www.apache.org/dev/openpgp.html
 - svn https://www.apache.org/dev/version-control.html#https-svn
 
+<h3>Preparing gpg key</h3>
+
+You can skip this section if you have been a release manager before.
+
+After generating the gpg key, you need to upload your key to a public key server. Please refer to
+<a href="https://www.apache.org/dev/openpgp.html#generate-key">https://www.apache.org/dev/openpgp.html#generate-key</a>
+for details.
+
+If you want to do the release on another machine, you can transfer your gpg key to that machine
+via the `gpg --export` and `gpg --import` commands.
+
+The last step is to update the KEYS file with your code signing key
+<a href="https://www.apache.org/dev/openpgp.html#export-public-key">https://www.apache.org/dev/openpgp.html#export-public-key</a>
+
+```
+# Move dev/ to release/ when the voting is completed. See Finalize the Release below
+svn co --depth=files "https://dist.apache.org/repos/dist/dev/spark" svn-spark
+# edit svn-spark/KEYS file
+svn ci --username $ASF_USERNAME --password "$ASF_PASSWORD" -m"Update KEYS"
+```
+
+<h3>Installing docker</h3>
```

Comment:
The thing I try to avoid is, maintain a document about how to set up the environment/dependencies. We should control it fully through the scripts.

I think we should not recommend people to do release without docker. If someone does want to run `do-release.sh` directly, he should read the docker file and other related scripts.

Reply:
OK, I see.